### DISCUSSION TITLE
Template trigger: for:

### DISCUSSION THREAD

**Discussion by chertvl** [home-assistant/frontend#12362] [Accessibility]
Hello :p
We able to use 'for:' function for template triggers in YAML.
Can we add this feature to GUI?

https://www.home-assistant.io/docs/automation/trigger/#template-trigger
<details>
  <summary>Screenshot</summary>
  
![Screenshot_20220420-150337_Home Assistant](https://user-images.githubusercontent.com/38353584/164227055-1e903281-d9cd-4c25-bfb8-c43154ca9fbf.jpg)

</details>

**Comment by frenck** [comment]
You can already use a template for the `for` parameter.

**Reply by chertvl** [reply]
Thank for your time, Frenck.
That not exactly what I mean.

Of course, we can use it with yaml-mode in GUI.
I mean it would be great to add input fields for time to template trigger gui, like we have it in state trigger or numeric state trigger.
Please, look at the screenshot at my topic.

Thanks!
p.s.: sorry for my English

**Reply by frenck** [reply]
Unmarked answer, you are right 👍
